Userland/Libraries/LibTest/Macros.h

```diff
@@ -38,10 +38,10 @@ void current_test_case_did_fail();
     } while (false)
 
 #undef TODO
-#define TODO()                                                                                   \
-    do {                                                                                         \
-        ::AK::warnln(stderr, "\033[31;1mFAIL\033[0m: {}:{}: TODO() called", __FILE__, __LINE__); \
-        ::abort();                                                                               \
+#define TODO()                                                                           \
+    do {                                                                                 \
+        ::AK::warnln("\033[31;1mFAIL\033[0m: {}:{}: TODO() called", __FILE__, __LINE__); \
```

Comment:
Other than this being redundant, did this cause an issue? 
this will `warnln` to `stderr` regardless, so you might have to change the commit message

Reply:
This was causing an error,
warnln does not take a destinations argument, which was causing a compilation error.

How would you describe the issue then

Reply:
Ah, that's right, warnln doesn't take a FILE*
Maybe "don't try to pass stderr to warnln"? I'm not sure, but "don't try to warnln to stdout" sounds like you're changing where warnln prints to

Reply:
Fixed that then its now `LibTest: Dont try to pass stderr to wanrln in TODO`

Reply:
uhhh...typo: `warnln`...

Reply:
ups...
fixed that....